src/components/BorrowerProfile/JumpLinks.vue

```diff
@@ -14,7 +14,7 @@
 			Borrower story
 		</a>
 		<a
-			class="tw-text-primary tw-pl-1 tw-pr-1 tw-font-medium tw-cursor-pointer"
+			class="tw-text-primary tw-pl-1 tw-pr-1 tw-font-medium tw-cursor-pointer tw-whitespace-nowrap"
```

Comment:
This is a small bug fix, this is the issue:
![Screen Shot 2022-04-04 at 2 51 06 PM](https://user-images.githubusercontent.com/4371888/162481906-66355909-ddf2-4cc5-86f4-e77e1be2c60d.png)